Source/WTF/wtf/PlatformHave.h

```diff
@@ -1478,8 +1478,9 @@
 
 #if (PLATFORM(MAC) && __MAC_OS_X_VERSION_MIN_REQUIRED >= 140000) \
     || ((PLATFORM(IOS) && !PLATFORM(IOS_FAMILY_SIMULATOR)) && __IPHONE_OS_VERSION_MIN_REQUIRED >= 170000) \
```

Comment:
Why can't we use `IOS_FAMILY` here instead of adding the `MACCATALYST` platform check?

Reply:
Good question! Mainly because we do NOT want it on for `tvOS`, and the version checks are different for the different platforms.

In the future, I hope we can move to `IOS_FAMILY`

Reply:
That makes sense; I think it might help to more explicitly spell this out with a code refactor or a comment. This was not clear upon initial reading.

Reply:
Keep in mind that IOS_FAMILY + a __IPHONE_OS_VERSION_MIN_REQUIRED check is pretty much never right, because the other members of IOS_FAMILY don't bump __IPHONE_OS_VERSION_MIN_REQUIRED (it's stuck at the version it was at when they forked off).

Reply:
Ah, I didn't realize that the other platforms set `__IPHONE_OS_VERSION_MIN_REQUIRED`.